aries-site/src/pages/learn/how-to-add-additional-controls-to-a-toolbar.mdx

```diff
@@ -0,0 +1,191 @@
+import { Box, Image } from 'grommet';
+
+## Prerequisites
+
+This guide builds off the ["How to add search and filter to DataTable" how-to guide](/learn/how-to-add-search-and-filter-to-datatable-with-data). While it's not necessary to complete that guide first, it may be helpful for additional context.
+
+Let’s get started!
+
+Use this [CodeSandbox template](https://codesandbox.io/p/devbox/how-to-add-additional-controls-to-a-toolbar-gn9q9j?file=%2Fsrc%2FApp.js%3A46%2C20) to start. The template is populated with a paginated DataTable configured to display SpaceX data with search and filter controls.
+
+## Importing the components
+
+In `App.js`, start by importing the Toolbar component and the Data controls you'll include in your toolbar.
+
+```
+import { Box, Data, DataTable, Grommet, DataFilters, DataSearch, DataSort, DataSummary, DataTableColumns, Toolbar } from "grommet";
+
+```
+
+## Compose the toolbar
+
+### Render the controls in the toolbar
+
+Remove the `toolbar` property from Data. Insert the Toolbar and Data controls above the DataTable.
```

Comment:
ToolBar component and what datacontrols? maybe add (DataSearch, DataSort, DataFilters)
also should we link to grommet docs

Reply:
Good idea.

Reply:
Done.